### PR TITLE
Rename package names

### PR DESCRIPTION
**Pull request by sivayogasubramanian**
Closes https://github.com/AY2122S1-CS2103T-W16-2/tp/issues/45.

Important note: https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/46#issuecomment-926317752

### CODE REVIEW

File: src/main/java/ay2122s1_cs2103t_w16_2/btbb/storage/UserPrefsStorage.java [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-import seedu.address.exception.DataConversionException;
-import seedu.address.model.ReadOnlyUserPrefs;
-import seedu.address.model.UserPrefs;
+import ay2122s1_cs2103t_w16_2.btbb.exception.DataConversionException;
+import ay2122s1_cs2103t_w16_2.btbb.model.ReadOnlyUserPrefs;
+import ay2122s1_cs2103t_w16_2.btbb.model.UserPrefs;
 
 /**
- * Represents a storage for {@link seedu.address.model.UserPrefs}.
+ * Represents a storage for {@link UserPrefs}.
```
Comment (EuniceSim142) [inline src/main/java/ay2122s1_cs2103t_w16_2/btbb/storage/UserPrefsStorage.java:12]: ay2122s1_cs2103t_w16_2.btbb wasn't added back
```suggestion
 * Represents a storage for {@link ay2122s1_cs2103t_w16_2.btbb.model.UserPrefs}.
```

File: src/test/java/ay2122s1_cs2103t_w16_2/btbb/logic/commands/AddClientCommandIntegrationTest.java [hunk 2]
```diff
@@ -43,7 +42,7 @@ public void execute_newClient_success() {
     public void execute_duplicateClient_throwsCommandException() {
         Client clientInList = model.getAddressBook().getClientList().get(0);
         ClientDescriptor clientInListDescriptor = new ClientDescriptorBuilder(clientInList).build();
-        assertCommandFailure(new AddClientCommand(clientInListDescriptor), model,
+        CommandTestUtil.assertCommandFailure(new AddClientCommand(clientInListDescriptor), model,
```
Comment (EuniceSim142) [inline src/test/java/ay2122s1_cs2103t_w16_2/btbb/logic/commands/AddClientCommandIntegrationTest.java:45]: Maybe import the assertCommandFailure method (like what's done for assertCommandSuccess),
then can change this to:
```suggestion
        assertCommandFailure(new AddClientCommand(clientInListDescriptor), model,
```

Comment (EuniceSim142) [inline src/test/java/ay2122s1_cs2103t_w16_2/btbb/logic/commands/AddClientCommandIntegrationTest.java:45]: Maybe import the assertCommandFailure method (like what's done for assertCommandSuccess),
then can change this to:
```suggestion
        assertCommandFailure(new AddClientCommand(clientInListDescriptor), model,
```

**Comment by zognin** [comment]
Some notes for this pr

- Not changing class names with AddressBook in this pr 
- Developers need to `rm preferences.json` and `rm -r data` after this is merged to allow new filename changes to populate

**Review by EuniceSim142 — COMMENTED** [review]
LGTM, have some minor nits.
There are some changes that can be made in DeveloperGuide.md (line 149), SettingUp.md (line 26) and Testing.md (line 32, 34, 36) but not sure if we should edit them yet.

**Review by EuniceSim142 — APPROVED** [review]

**Pull request merged**